Automattic/wp-calypso
# GS on Personal A/B: Stop upselling Personal to `treatment` users

Part of https://github.com/Automattic/dotcom-forge/issues/3062

## Proposed Changes

After finishing the GS on Personal experiment, we decided to stick with `control` (keep Global Styles in the Premium plan, see paYJgx-3PM-p2#comment-4032), so this PR removes the `treatment` upsell experience and always upsells the Premium plan.

Participants who were assigned to `treatment` will continue getting Global Styles on the Personal plan, but the UI will always upsell the Premium plan.

Section | Before | After
--- | --- | ---
Plans grid | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/d9f1ece0-9697-4913-966d-a04c6d3ac2df) | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/8b8992cd-bd91-4c8d-a829-32bb058c4326)
Design picker | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/03133c24-b7e0-4b26-adc5-c0e9a3e97882) | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/b50fa35f-af5f-4bc7-90b7-7dbc654401c4)
Design preview | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/0328cf4c-6ca7-4a81-9f40-4dbf4a3c9e2d) | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/8299899c-286d-4057-b82c-5015227e5751)
Site editor | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/e341151f-df89-4378-8175-385b6fb9d04f) | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/7ba95558-364c-4172-ac17-8340e8cd177b)
Site editor | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/38a8c2bf-81c9-40b5-b68d-ce134716c153) | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/8e19bce8-332e-4224-a66b-45f3f908aafb)
Frontend | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/978f4d66-5e46-47bc-82a9-feda2bc134f8) | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/52b2c5db-74fc-4560-9fd6-bb2168c181c4)
Site assembler | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/28196180-d256-47ea-a051-c7e79c26b5f8) | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/4ba9a849-4f13-4caa-a2c3-7c17f3888993)
Privacy settings | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/4a41737e-2e80-4a6d-9e22-6a26d18f285c) | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/86fb7a0b-6b6a-4aea-b047-66f3a49c3c09)
Theme showcase | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/3909db32-f1fe-45cc-ad18-2255807d385e) | ![image](https://github.com/Automattic/wp-calypso/assets/1233880/bff65011-be84-4884-b335-e9b3b1401a22)

## Testing Instructions

- Assign yourself to the `treatment` variant of the experiment (pbxNRc-2HR-p2)
- Apply these changes to your sandbox
- Sandbox a site with Limited Global Styles
- Use the Calypso live link below
- Check that we're now upselling the Premium plan in all the sections listed above
